site/visitor-console/src/config/getAmplifyConfig.tsx

```diff
@@ -0,0 +1,28 @@
+// getAmplifyConfig.ts
+
+interface AmplifyConfig {
+  userPoolId: string;
+  userPoolClientId: string;
+}
+
+export function getAmplifyConfig(): AmplifyConfig {
+  const hostname = window.location.hostname;
+
+  if (hostname.includes("visit.cumaker.space")) {

```

Comment:
this check here will return true if the domain is `beta-visit.cumaker.space`, as it includes the prod url in it. I would just exact compare it to the domain we expect

Reply:
True, good catch